assets/superformula-logo.svg

```diff
@@ -0,0 +1,5 @@
+<svg fill="white" version="1.1" id="Layer_1" x="0px" y="0px" viewBox="0 0 28 40" style="enable-background:new 0 0 28 40" xml:space="preserve"><path d="M26.7,13c-0.2-0.1-0.5,0-0.7,0.1L14.2,24.8v-11L27,1.2c0.2-0.2,0.2-0.5,0.1-0.7C27,0.2,26.8,0,26.5,0H13.5
+	c-0.1,0-0.3,0.1-0.4,0.2c0,0,0,0,0,0L0.2,13.1c0,0-0.1,0.1-0.1,0.1C0,13.3,0,13.4,0,13.6c0,0.1,0,0.1,0,0.2c0,0.1,0,0.1,0.1,0.2
+	c0,0,0.1,0.1,0.1,0.1l12.4,12.4L0.2,38.8C0,39,0,39.3,0,39.6C0.2,39.8,0.4,40,0.7,40h12.9c0.2,0,0.3-0.1,0.4-0.2l12.7-12.7
+	c0.2-0.1,0.4-0.3,0.4-0.6V13.6C27.2,13.3,27,13.1,26.7,13z M14.2,1.3h10.6L14.2,12V1.3z M12.9,38.7H2.3L12.9,28V38.7z M12.9,24.8
+	L2.3,14.2h10.6V24.8z M12.9,12.9H2.3L12.9,2.3V12.9z M14.2,37.7V27.2h10.5L14.2,37.7z M25.8,25.8H15.3l10.5-10.5V25.8z"></path></svg>
```

Comment:
info:  nice touch 😅

Reply:
Thanks. Always try to deliver the "One more thing...". 😃